content/javascript/concepts/objects/terms/defineProperties/defineProperties.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: '.defineProperies()'
+Description: 'Defines new or modify existing properties directly on an object'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Properties'
+  - 'Objects'
+  - 'Functions'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `Object.defineProperties()` is used to define new or modify existing properties directly on an object or to add multiple properties to an object at once. This method is often used for property descriptors and fine-grained control over object properties.
+
+## Syntax
+
+```pseudo
+Object.defineProperties(object, descriptors)
+```
+
+- `object`: The object on which to define or modify it's properties.
+- `descriptors`: An object where each key represents a property name, and the corresponding value is the property descriptor.
+
+## Example
+
+In this example `Object.defineProperties` is used to define two properties (`property1` and `property2`) on the `myObject` object. Each property has a descriptor object specifying attributes like `value` and `writable`.
+The `value` attribute is the value of the property and the `writable` attribute determines whether the property can be modified.
```

Comment:
```suggestion
```
This sentence won't be necessary as we are adding that in the syntax section now.